ios/browser/policy/sources.gni

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+brave_browser_policy_sources =
+    [ "//brave/ios/browser/policy/brave_simple_policy_map_ios.h" ]
+
+brave_browser_policy_deps =
+    [ "//brave/components/brave_wallet/common:pref_names" ]
```

Comment:
This setup matches desktop, but perhaps it'd be better instead just have a single dep (`//brave/ios/browser/policy`) and no sources so that we dont need to continuously add sources to the Chrome target